styletesla.css

```diff
@@ -1,97 +1,751 @@
+    background-position-y: 20%;
+    background-size: contain;
+  }
+
+  .description {
+    width: 100vw;
+    margin-bottom: 35px;
+  }
+
+  .speed {
+    width: 160px;
+    margin-right: 20px;
+  }
+
+  .road {
+    margin-right: 50px;
+  }
+
+  .icon_speed {
+    width: 38px;
+    height: 28px;
+  }
+
+  .icon_road {
+    width: 22px;
+    height: 35px;
+  }
+
+  .number {
+    font: 27px/20px montserratsemibold;
+  }
+
+  .dimension /*.secunda, .km*/ {
+    font: 15px/14px montserratbold;
+  }
+
+  .text_bottom {
+    font: 10px/10px montserratregular;
+   }
+
+  .icon_redwheel {
+    width: 22px;
+    height: 22px;
+   }
+
+  .wheel__desc {
+    font: 13px/16px montserratbold;
+  }
+
+  .btn__order {
+    width: 160px;
+    height: 40px;
+    font: 13px/16px montserratbold;
+  }
+}
+
+@media (max-width: 370px) {
+  .speed {
+    width: 122px;
+  }
+
+  .wheel {
+    margin-right: 25px;
+  }
+
+  .btn__order {
+    width: 140px;
+  }
+}
+
+ /*вторая секция*/
 .advantage {
-	height: 100vh;
-	background: url('img/rulj.png') bottom left no-repeat; 
-	display: flex;
-	justify-content: space-between;
+  width: 100%;
+  background: url('img/rulj.png') left bottom no-repeat;
+  background-size: contain;
+  display: flex;
+  justify-content: space-between;
+}
 
+.advantage__title {
+  font: 70px/85px montserratsemibold;
+  padding-left: 77px;
+  color: #e51937;
 }
+
 .sidebar {
-	height: 100vh;
-	width: 30%;
-	right: 0;
-	background:#E51937;
+  height: 100%;
+  width: 28%;
+  right: 0;
+  background:#e51937;
+  padding-left: 50px;
+  color: white;
+}
 
+.sidebar__title {
+  font: 30px/35px montserratsemibold;
+  padding-top: 3vh;
+  margin: 0;
+  margin-bottom: 5px;
 }
-.sidebar__title, .sidebar__discription {
-	color:white;
+
+.sidebar__discription {
+  font: 15px/18px montserratmedium;
 }
-.sidebar__title {
-	padding-top: 51px;
-	margin: 0;
+
+.list {
+  list-style: none;
+  padding-left: 0;
+  margin-bottom: 170px;
+}
+
+.list p {
+  margin: 15px;
 }
+
+.icon_dujm,
+.icon_po,
+.icon_litr {
+  display: flex;
+  align-items: center;
+}
+
+.icon_screen {
+  width: 36px;
+  height: 53px;
+}
+
+.icon_wifi {
+  width: 44px;
+  height: 34px;
+}
+
+.icon_bag {
+  width: 36px;
+  height: 52px;
+}
+
+.po {
+  font: 20px/18px montserratsemibold;
+}
+
+.button {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 70px;
+}
+
+.button__more {
+  display: flex;
+  align-items: center;
+}
+
+.more__sign {
+  width: 22px;
+  height: 22px;
+  margin-right: 5px;
+}
+
+.know {
+  font: 13px/16px montserratbold;
+  cursor: pointer;
+}
+
+.button__order {
+  border: 2px solid white;
+  color: white;
+  background: #e51937;
+  font: 13px/16px montserratbold;
+  text-transform: uppercase;
+  margin-right: 30px;
+}
+
+/*планшет*/
+@media (max-width: 1270px) {
+  .sidebar {
+    width: 35%;
+  }
+}
+
+@media (max-width: 1165px) {
+  .advantage {
+    height: 900px;
+    width: 100%;
+    background: url('img/rulj-planshet.png') top no-repeat; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)